js/remote/index.js

```diff
@@ -160,12 +183,31 @@ function startMagStream(data) {
 
     // respond anyway cuz they all do that
     const response = new responseToken({}, data.headers.transactionID);
-    _client.write(response.stringify());
+    sendToken(response);
 }
 
 function stopMagStream(data) {
     clearInterval(_magInterval);
 
     const response = new responseToken({}, data.headers.transactionID);
-    _client.write(response.stringify());
+    sendToken(response.stringify());
+}
+
+function consumeControllerData(data) {
+    if (args.debug) {
+        // do nothing if the server is running in debug mode
+        const response = new responseToken({}, data.headers.transactionID);
+        sendToken(response);
+        return;
+    }
+}
+
+function sendToken(token) {
+    /*
+     * WARN-JOB:
+     * uncommenting the console.log line below will make the server crash after a few seconds
+     * if you try deploying using the --local tag
+     */
+    // console.log('sending it:' + token.stringify());
```

Comment:
Given the warning, why does this still exist? I feel like it would be worth either removing, or uncommenting with a check for `--local`.

Reply:
This is mostly for debugging, I'm just gonna leave it in there for now. I'm working on a branch that's gonna use a real logger and hopefully we can figure out a better solution there.

Reply:
see [CALICAWESOME/better-logging](https://github.com/Wentworth-IEEE/rovsquad18/tree/better-logging)